BB-SC/MWLayer/Configuration/BB_OTA_Master.md

```diff
@@ -0,0 +1,78 @@
+
+# OTA Master
+
+## BB Tags(s)
+<!-- Tag(s) define in which area(s) (cloud, in-vehicle) the BB is executed, and what type of BB it is (tool, process, microservice) -->
+BB-SC
+
+## Functional Clusters
+<!-- In which Functional Cluster the BB be located; if none of the existing fit new required -->
+Configuration
+
+## Layer
+<!-- AppLayer, MWLayer, OSLayer, HWLayer -->
+MWLayer
+
+## Known Implementation
+
+## ID (unique name)
+
+## Description
+<!-- General Description of the BB -->
+The OTA master is a central SWC to deploy OTA SW updates within the vehicle network. Besides the basic functionality of SW installation, it implements the necessary failsafe or failover behavior to ensure the robustness of the update process (such as backup and rollover mechanism). In addition, it must manage and verify the integrity of the installed SW after it has been transferred from the backend, potentially even before installing the SW. It also coordinates local OTA salves distributed in the vehicle, by self-contained interface and command control. As a prerequisite for a successful update, the update master has also to be capable to read the vehicle configuration and control reprogramming history.
+
```

Comment:
Depending on which local context this applies to (where should the core orchestrator live?), there are existing implementations for this.

AUTOSAR: UCS - but not OSS-usable
higher-level: Eclipse Ankaios, Eclipse Kanto, Eclipse Symphony, Eclipse BlueChi

Reply:
Agreed. Will be taken care of in the next pull request. will be linked in Known Implementations